steps/bitrise-step-eschecker/0.0.1/step.yml

```diff
@@ -0,0 +1,55 @@
+title: esChecker, by eShard
+summary: |
+  Run Mobile Application Security Testing using esChecker
+description: |
+  Run esChecker analyses on new application builds directly from Bitrise.
+website: https://github.com/eshard/bitrise-step-eschecker
+source_code_url: https://github.com/eshard/bitrise-step-eschecker
+support_url: https://github.com/eshard/bitrise-step-eschecker/issues
+published_at: 2022-12-02T18:01:31.197476+01:00
+source:
+  git: https://github.com/eshard/bitrise-step-eschecker.git
+  commit: cfa105125ac67168579a86f39b013e1bd195110b
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- opts:
+    description: |
+      Path to the binary file to be tested
+    is_required: true
+    summary: The app file you want to test.
+    title: A Bitrise generated APK or IPA path
```

Comment:
Just out of curiosity, does this accept an Android AAB too?

Reply:
No, not yet. APK only for now.